src/cs/tool/Program.cs

```diff
@@ -18,12 +18,32 @@ static unsafe void Main(string[] args)
         {
             // This code lets us pass in an argument of where to search for the library at.
             // Very helpful for testing
-            if (args.Length > 0)
+            if (args.Length == 0)
+            {
+                Console.WriteLine("Usage: MsQuicTool <DomainName> [PathToMsQuic]");
+                return;
+            }
+
+            string DomainName = args[0];
+
+            if (string.IsNullOrWhiteSpace(DomainName))
+            {
+                Console.WriteLine("DomainName cannot be empty.");
+                return;
+            }
+
+            if (DomainName.Length > 253)
```

Comment:
Is there a reason for "253" here?
Below, when we convert it to bytes, we have a buffer of 50 bytes. Wont it be too short for any input of more than 25 characters accepted here?